sceptre/diffing/stack_differ.py

```diff
@@ -201,6 +201,9 @@ def _create_deployed_stack_config(
             if err.response["Error"]["Message"].endswith("does not exist"):
                 return None
 
+            # Unknown error, raise it as-is
+            raise err
```

Comment:
The PR subject says resolves #1520, but isn't it actually resolving #1519?

Also the idea in #1519 is to make the permissions error more clear to the user.  Would just re-raising the exception be clear enough?

Reply:
@zaro0508 If Dan's not around, I don't have any immediate plans to implement centralised exception handling and I was not entirely convinced that we should be trying to provide error messages more helpful than whatever the client tells you. 

I don't however think that discussion should block us fixing the actual bug though, thus this PR.

Reply:
ok, can you address my question regarding #1520 vs #1519?

Reply:
That seems to be a typo that Dan did, and I've just taken over his branch. I have updated the description here to refer to #1519.

Reply:
Can we get this one approved now @zaro0508